packages/playwright-core/src/cli/cli.ts

```diff
@@ -298,6 +298,8 @@ program
 program
     .command('show-trace [trace...]')
     .option('-b, --browser <browserType>', 'browser to use, one of cr, chromium, ff, firefox, wk, webkit', 'chromium')
+    .option('-h, --host <host>', 'Host to serve trace on', 'localhost')
+    .option('-p, --port <port>', 'Port to serve trace on', '9323')
```

Comment:
This is a different value from what it used to be.

Reply:
At `9322` now